packages/account/src/Sections/Security/ClosingAccount/closing-account-pending-conditions/closing-account-pending-balance.tsx

```diff
@@ -19,7 +19,7 @@ type TClosingAccountPendingBalanceProps = {
     account_balance: TDetailsOfMT5Account[] | TDetailsOfDerivXAccount[];
 };
 
-type TShortcode = Parameters<typeof getCFDAccountDisplay>[0]['shortcode'];
+type TShortcode = Exclude<TDetailsOfMT5Account['landing_company_short'], 'seychelles'>;
```

Comment:
why do we manually remove 'seychelles'

Reply:
<img width="1157" alt="Screenshot 2024-07-04 at 3 52 53 PM" src="https://github.com/binary-com/deriv-app/assets/121229483/3d34b5ab-0b75-402d-91c8-26fad11966d8">

Hi @amina-deriv its because we are having error here.